packages/features/ee/teams/pages/team-members-view.tsx

```diff
@@ -203,6 +203,7 @@ const MembersView = () => {
             <MemberInvitationModal
               isPending={inviteMemberMutation.isPending}
               isOpen={showMemberInvitationModal}
+              justEmailInvites={true}
```

Comment:
The goal of this issue is to fix errors when using a username to invite and not limit the invitation to an email address. Also, do not commit the lock file when there are no changes in package.json